src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -300,7 +298,7 @@ protected void onPostExecute(Void result) {
       protected void executeAction(@Nullable Void parameter) {
         for (long chatId : selectedConversations) {
           if (chatId == DcChat.DC_CHAT_ID_DEADDROP) {
-            dcContext.marknoticedContact(getListAdapter().getDeaddropContactId());
+            dcContext.marknoticedChat(DcChat.DC_CHAT_ID_DEADDROP);
```

Comment:
`dc_marknoticed_chat(DC_CHAT_ID_DEADDROP)` won't work, it would just be a no-op as core the [corresponding sql-statement](https://github.com/deltachat/deltachat-core-rust/blob/master/src/chat.rs#L1944) works with real-chat-ids.

i think, the idea of this handling was to "move the deaddrop away" when the user taps "archive" for it, not sure, however, if that is nice ux (there is no "archive" for the deaddrop)